dahu/core/app/scripts/views/workspace/objects/tooltip.js

```diff
@@ -76,6 +81,15 @@ define([
 
         onRender: function() {
             this.updateColor();
+        },
+
+        onScaleChange: function(scale) {
```

Comment:
according to https://github.com/marionettejs/backbone.marionette/blob/master/docs/marionette.behavior.md#the-event-proxy you can remove this function since events are already propagated to behaviors ;)